sentry_sdk/_types.py

```diff
@@ -87,3 +90,56 @@
     MeasurementUnit = Union[DurationUnit, InformationUnit, FractionUnit, str]
 
     ProfilerMode = Literal["sleep", "thread", "gevent", "unknown"]
+
+    # Unit of the metrics.
+    MetricUnit = Literal[
+        "none",
+        "nanosecond",
+        "microsecond",
+        "millisecond",
+        "second",
+        "minute",
+        "hour",
+        "day",
+        "week",
+        "bit",
+        "byte",
+        "kilobyte",
+        "kibibyte",
+        "mebibyte",
+        "gigabyte",
+        "terabyte",
+        "tebibyte",
+        "petabyte",
+        "pebibyte",
+        "exabyte",
+        "exbibyte",
+        "ratio",
+        "percent",
+    ]
+
+    # Type of the metric.
+    MetricType = Literal["d", "s", "g", "c"]
+
+    # Value of the metric.
+    MetricValue = Union[int, float, str]
+
+    # Internal representation of tags as a tuple of tuples (this is done in order to allow for the same key to exist
+    # multiple times).
+    MetricTagsInternal = Tuple[Tuple[str, str], ...]
+
+    # External representation of tags as a dictionary.
+    MetricTagValue = Union[
+        str,
+        int,
+        float,
```

Comment:
I changed this intentionally because the current sentry usage allows floats and ints. I would like to remove floats but probably want to remove that restriction later. In general if you are not using typing, then you can already pass anything since it undergoes stringification.